support-frontend/assets/helpers/legacyTypeConversions.ts

```diff
@@ -33,10 +33,6 @@ export const getLegacyProductType = (
 			throw new Error(
 				'OneTimeContribution does not exist in the old product catalog',
 			);
-		case 'GuardianPatron':
```

Comment:
I've removed GuardianPatron from the list of active products because we don't sell it through this site. 

I think we could remove OneTimeContribution as well because we don't use any catalog information about it, but as we do sell it it does seem slightly odd not to have it in the ActiveProducts list. What does anyone else think?

Reply:
Think I would keep it. `OneTimeCheckout` still a valid `ActiveProductKey` that needs convertig to a `LegacyProductType` and throwing an error seems a good way to handle to me